python/nano/scripts/bigdl-nano-init

```diff
@@ -55,6 +55,8 @@ TCMALLOC=1
 # Set TF_ENABLE_ONEDNN_OPTS
 export TF_ENABLE_ONEDNN_OPTS=1
 
+OPTIND=1
```

Comment:
`OPTIND` might not be 1 since this is value from the current shell when we use `source bigdl-nano-init`